paper.md

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
This file and the bib file should go in a folder called josspaper, or something like that

Reply:
I agree we could do it this way but several examples have also done it the way I did it here

Reply:
If we can do it either way, then we should put it in a folder.